src/components/plots/PlotHeader.jsx

```diff
@@ -3,27 +3,21 @@ import React, {
   useEffect, useCallback, useState,
 } from 'react';
 import useSWR from 'swr';
-import {
-  PageHeader, Row, Col, Button, Skeleton, Space,
-} from 'antd';
+import { Button, Skeleton, Space } from 'antd';
 import { useSelector, useDispatch } from 'react-redux';
 import { useRouter } from 'next/router';
 import PropTypes from 'prop-types';
 import moment from 'moment';
 import { useBeforeunload } from 'react-beforeunload';
+import Header from 'components/Header';
 import FeedbackButton from '../FeedbackButton';
 import ReferralButton from '../ReferralButton';
 import { savePlotConfig } from '../../redux/actions/componentConfig/index';
-import itemRender from '../../utils/renderBreadcrumbLinks';
 import { getFromApiExpectOK } from '../../utils/getDataExpectOK';
 import { LOAD_CONFIG } from '../../redux/actionTypes/componentConfig';
 import { initialPlotConfigStates } from '../../redux/reducers/componentConfig/initialState';
 
-const Header = (props) => {
-  const {
-    experimentId, plotUuid, finalRoute,
-  } = props;
-
+const PlotHeader = ({ title, experimentId, plotUuid }) => {
```

Comment:
Do you recommend we start using this new format to restructure props?

Reply:
Well not really, I just saw it somewhere and reused it. If there are not many properties looks good but I don't have a strong opinion.